dll/cpl/powercfg/lang/cs-CZ.rc

```diff
@@ -2,7 +2,7 @@
  * PROJECT:     ReactOS Power Configuration Applet
  * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Czech resource file
- * TRANSLATORS: Copyright 2007 Kamil Hornicek <kamil.hornicek@reactos.org>
+ * TRANSLATORS: Copyright 2007 Kamil Horníček <kamil.hornicek@reactos.org>
```

Comment:
Hmm... more than one letter changed?

- Horníček: [not found](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Horn%C3%AD%C4%8Dek) ⚠️ 
- Hornícek: [two occurences](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Horn%C3%ADcek)

Reply:
The form "Horníček" is a standard Czech surname supposed on the basis of its diacritics-free form "Hornicek". The form "Hornícek" seems to be ill-written - somewhere between the two previously mentioned forms.

One can eventually check the prevalence of surnames in the Czech Republic on https://www.kdejsme.cz/seznam/prijmeni/H (it is based on statistics from 2016 provided by Czech Interior Ministry).

Reply:
Ok we will have to update it everywhere then I guess 👀

Reply:
Or we could stick with the form without diacritics as it was ok until now, and change it to the form with diacritics at newer versions only (if we should change it at all),

Reply:
Just my 2 cents: I do prefer the version without the diacritics, especially in source-files to increase the chance of keeping them ANSI. Contrary In *.rc files it's not really that important, because most of them require to be UTF8 anyway. But here too, in case of uncertainty strip the diacritics.

Reply:
So I would suggest accepting names with diacritics and preserving them in resource files, but not requiring them to be rewritten with diacritics if they were originally stated without them (either in the code or in the commit metadata).

Reply:
The fixed name is indeed the correct one with correct diacritics:
https://reactos.org/wiki/User:Preston
> full name: Kamil Horníček